resources/function_help/json/if

```diff
@@ -4,10 +4,11 @@
   "description": "Tests a condition and returns a different result depending on the conditional check.",
   "arguments": [
 	{"arg":"condition","description":"the condition which should be checked"},
-	{"arg":"result_when_true","description":"the result which will be returned when the condition is True"},
-	{"arg":"result_when_false","description":"the result which will be returned when the condition is False"}
+	{"arg":"result_when_true","description":"the result which will be returned when the condition returns True or not null value"},
```

Comment:
ok.
How does indentation works in json? with the chosen description, I end up with a very long line I'd like to split for convenient reading. To what should I align next line? Or should I keep one line by item?